<?xml version="1.0"?>

<launch>

  <node pkg="traj_generator" type="circle_test" name="circle_test" output="screen">
    <param name="radius" type="double" value="1.0"/>
    <param name="period" type="double" value="6.0"/>
    <param name="rev" type="double" value="1.0"/>
    <param name="follow_yaw" type="bool" value="false"/>
    <param name="move_z" type="bool" value="true"/>
    <param name="z_diff" type="double" value="0.3"/>
  </node>

</launch>
